### PR TITLE
Improve manual install experience.

### PR DESCRIPTION
**Pull request by markstos**
- Add `make install` target
  - Document manual installation steps
  - Mention Arch Linux package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,20 @@ it just exits when it changes. This is intentional -- X11's selection API is
...
 clipnotify.
+
+## Installation via Packages
+
+On Arch Linux, you can install the `clipnotify` package.
+
+## Manual installation
+
+First you need to make sure "XFixes.h" is already installed. On Debian-derived systems,
+this is located in the package `libxfixes-dev`
```
Comment (cdown) [inline README.md]: s/$/./

File: README.md [hunk 2]
```diff
@@ -16,3 +16,20 @@ it just exits when it changes. This is intentional -- X11's selection API is
...
+
+
+    git clone https://github.com/cdown/clipnotify.git
+    cd clipnotify
+    make
+    sudo make install
+
+The final step will install the `clipnotify` binary at `/usr/bin/clipnotify`
```
Comment (cdown) [inline README.md]: ditto

**Review by cdown — COMMENTED** [review]
Looks fine, thanks. Just a couple of cleanups.

**Comment by markstos** [comment]
Cleaned up commit pushed.

**Comment by iamleot** [comment]
JFTR, at least the `install:` target was implemented differently via #12.

**Comment by cdown** [comment]
The readme change is still fine though, although it probably shouldn't explicitly call out Debian-specific stuff.

**Pull request closed without merging**